BEIMA.Backend/Startup.cs

```diff
@@ -1,27 +0,0 @@
-﻿using BEIMA.Backend.StorageService;
-using Microsoft.Azure.Functions.Extensions.DependencyInjection;
-using Microsoft.Extensions.DependencyInjection;
-using System;
-
-[assembly: FunctionsStartup(typeof(BEIMA.Backend.Startup))]
-
-namespace BEIMA.Backend
-{
-    public class Startup : FunctionsStartup
```

Comment:
Why don't we need this anymore?

Reply:
This was a remnant of how I had the storage service setup to use dependency injection. Thought I removed it previously, but I guess not.